packages/@ama-sdk/schematics/schematics/typescript/core/index.spec.ts

```diff
@@ -32,7 +33,7 @@ describe('Typescript Core Generator', () => {
     const tree = await runner.runSchematic('typescript-core', {
       specPath: path.join(__dirname, '..', '..', '..', 'testing', 'MOCK_swagger.yaml')
     }, baseTree);
-    const content: any = tree.readJson('/openapitools.json');
+    const content = tree.readJson('/openapitools.json') as any as OpenApiToolsConfiguration;
```

Comment:
not sure why `as any` is needed here :S

Reply:
the output of `readJson` doesn't seem to like `undefined` fields, I had to change them to `null` to make the interface compatible without the use of `any`